tests/hmm/run.sh

```diff
@@ -12,9 +12,15 @@ export MAXCHUNKS=1
 #As long as one person produces it, other people can run the analysis on this
 #Currently, use the cache provided by Joosep
 #export CACHE_PATH=/storage/user/jpata/hmm/cache
+<<<<<<< HEAD
```

Comment:
oops, looks like the merge conflicts got in...

Reply:
Sorry, removed it